src/k8s/pkg/k8sd/features/localpv/localpv_test.go

```diff
@@ -0,0 +1,155 @@
+package localpv_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	. "github.com/onsi/gomega"
+	"k8s.io/utils/ptr"
+
+	"github.com/canonical/k8s/pkg/client/helm"
+	helmmock "github.com/canonical/k8s/pkg/client/helm/mock"
+	"github.com/canonical/k8s/pkg/k8sd/features/localpv"
+	"github.com/canonical/k8s/pkg/k8sd/types"
+	snapmock "github.com/canonical/k8s/pkg/snap/mock"
+)
+
+func TestDisabled(t *testing.T) {
```

Comment:
I believe since we're in the `k8sd/features/localpv_test` package, it's better to keep the name as simple as possible according to [Go naming conventions]. Since we don't have any other feature to test here. I agree with you in case of something like `features/cilium/...` since there we're dealing with network and loadbalancer tests and need to be more specific. wdyt?

Reply:
That is fine by me, my thought is that for us it is obvious that the test is about the feature being disabled or enabled but for people new to our code base it may not. I think we should be okay though since people unfamiliar will quickly pick it up by reading the test.